pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -205,7 +204,11 @@ func (r *ReconcileKnativeServing) install(instance *servingv1alpha1.KnativeServi
 		}
 	}
 	if err != nil {
-		instance.Status.MarkInstallFailed(err.Error())
+		if _, ok := err.(*common.NotYetReadyError); ok {
+			instance.Status.MarkInstallFailed("Install in progress: " + err.Error())
```

Comment:
Let's add a `MarkInstallNotReady` function and call that here. Having the CR throw an `Error` even though it'll fix itself is not nice.

Reply:
Sure 
added like below
```
func (is *KnativeServingStatus) MarkInstallNotReady(msg string) {
	conditions.Manage(is).MarkFalse(
		InstallSucceeded,
		"NotReady",
		msg)
}
```